README.md

```diff
@@ -4,14 +4,44 @@ This repository contains a set of libraries designed to support the use of
 USD in C#/Unity. The goal of this package is to make it maximally easy to
 integrate and explore Universal Scene Description.
 
+![USD screenshot](Images/USD_global_screenshot.png)
+
+
 # Getting Started
 
 To get started, install USD via the Unity Package manager, either by using
 an official repository, or by browsing for a local package while working with
 source.
 
 Once the USD package is installed, a USD menu will appear, enabling you to
-easily import an export USD files.
+easily import and export USD files.
+
+![USD menu](Images/USD_menu.png)
+
+
+## Requirements
+
+* Windows / OSX
+* Unity version: 2018.3 (2018.3.4f1 and up) and 2019.1 (2019.1.0b2 and up)   
```

Comment:
I would only mention 2018.3.4f1 and up. Second part redundant ?

Reply:
2019.1 alphas and first beta will not work as the timeline package is either not here yet or with a version that not `1.0.0` yet.

Reply:
Makes sense :)